### PR TITLE
Use strings.EqualFold in case-insensitive comparisons

### PR DESCRIPTION
**Pull request by apocelipes**
`strings.EqualFold` is simpler and much faster than `strings.ToLower`.

### CODE REVIEW

**Comment by apocelipes** [comment]
Irrelative failure, seems like 624d6c571cce's code ran into some problems.

File: processor/processor.go [hunk 1]
```diff
@@ -332,7 +332,7 @@ func setupCountAs() {
 			// See if we can identify based on language name which is the most
 			// reliable as the name should be unique
 			for name := range languageDatabase {
-				if strings.ToLower(name) == strings.ToLower(t[1]) {
+				if strings.EqualFold(name, t[1]) {
```
Comment (ccoVeille) [inline processor/processor.go:335]: I fixed this one in #478. I can remove it from my PR. Not a problem

Comment (apocelipes) [reply processor/processor.go:335]: I apologize for not noticing that this part duplicates your PR. Also thank you for your generosity in transferring these two lines of changes to me.

Comment (ccoVeille) [reply processor/processor.go:335]: I removed it from my PR

Comment (apocelipes) [reply processor/processor.go:335]: Thank you :)

**Comment by boyter** [comment]
Neat. One of those ones that I am sure had I written it today would have picked up. Thanks.

**Pull request merged**